app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with model: recipe, local: true do |f| %>
+    <div class="new_recipe_form">
+    <h1>New Recipe</h1>
+    <div class="recipe-field">
+        <%= f.label :name %>
+        <%= f.text_field :name %>
+    </div>
+    <div class="recipe-field">
+        <%= f.label :preparation_time %>
+        <%= f.number_field :preparation_time %>
+    </div>
+    <div class="recipe-field">
+        <%= f.label :cooking_time %>
+        <%= f.number_field  :cooking_time %>
+    </div>
+    <div class="recipe-field">
+        <%= f.label :description %>
+        <%= f.text_area :description %>
+    </div>
+    <div class="field">
+    <%= f.label :public %>
+     <%= f.check_box :public %><br />
+
+   </div>
+   <%= f.submit "Create Recipe", class: "recipe-btn" %>
+    </div>
+    <% end %>
```

Comment:
- I have a validation error when I try creating a Recipe. This error only goes away when I take away the validation in the `Recipe` Model. Please look into this and make sure your app works properly 🙏 
![jryjryj](https://user-images.githubusercontent.com/73607512/236637477-73160c28-fcb2-4a74-a86d-23f433ce9fcf.PNG)

Reply:
Hi @mckent05 ,
Thank you for the review
We are not sure why you are getting that error because on our side it works
<img width="795" alt="Screen Shot 2023-05-06 at 9 18 53 PM" src="https://user-images.githubusercontent.com/75949338/236641122-b0ab141e-7897-4368-9056-5067c5315c58.png">